packages/webapp/src/routes/credits-estimation/credits-individual-or-organisation.js

```diff
@@ -0,0 +1,12 @@
+import constants from '../../credits/constants.js'
+
+export default [
+  {
+    method: 'GET',
+    path: constants.routes.ESTIMATOR_CREDITS_INDIVIDUAL_ORG,
+    options: {
+      auth: false
+    },
+    handler: (_request, h) => h.view(constants.views.ESTIMATOR_CREDITS_INDIVIDUAL_ORG)
+  }
+]
```

Comment:
could we please get code/file format look like land routes to make it easier for someone new to follow and future maintenance